### PR TITLE
add test id 22 Details Restaurant

### PR DESCRIPTION
**Pull request by nik-trifonov**

### CODE REVIEW

File: src/test/java/com/easyrestaurant/tests/UnauthorizedUserDetailsRestaurant.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import org.testng.annotations.Test;
+
+public class UnauthorizedUserDetailsRestaurant {
+	
+	@Test
+	public void Tested1() {
+		System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver");
+		WebDriver driver= new ChromeDriver();
+		
+		sleep(3000);
```
Comment (desislavavelichkova) [inline src/test/java/com/easyrestaurant/tests/UnauthorizedUserDetailsRestaurant.java:19]: Better to use implicitly wait.

**Review by desislavavelichkova — APPROVED** [review]
Everything work properly.

**Review by TurgayArif — APPROVED** [review]
Please implement page object model for this test.

File: src/test/java/com/easyrestaurant/tests/UnauthorizedUserDetailsRestaurant.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
+package selenium;
```
Comment (TurgayArif) [inline src/test/java/com/easyrestaurant/tests/UnauthorizedUserDetailsRestaurant.java:1]: Please fix the package.

**Pull request merged**